cedar-policy-validator/src/schema/test_579.rs

```diff
@@ -0,0 +1,1431 @@
+        &ExpectedErrorMessageBuilder::error("failed to resolve type: NS2::MyType")
+            .help("`NS2::MyType` has not been declared")
+            // TODO: why don't we have an underline here? (Uncommenting this produces test failure)
+            //.exactly_one_underline("NS2::MyType")
+            .build(),
+    );
+}
+#[test]
+fn B3a2_human() {
+    assert_parse_error_human(
+        &a2_human(B3_human()),
+        &ExpectedErrorMessageBuilder::error("failed to resolve type: NS2::MyType")
+            .help("`NS2::MyType` has not been declared")
+            // TODO: why don't we have an underline here? (Uncommenting this produces test failure)
+            //.exactly_one_underline("NS2::MyType")
+            .build(),
+    );
+}
+#[test]
+fn B3b1_human() {
+    assert_parse_error_human(
+        &b1_human(B3_human()),
+        &ExpectedErrorMessageBuilder::error("failed to resolve type: NS2::MyType")
+            .help("`NS2::MyType` has not been declared")
+            // TODO: why don't we have an underline here? (Uncommenting this produces test failure)
+            //.exactly_one_underline("NS2::MyType")
+            .build(),
+    );
+}
+#[test]
+fn B3b2_human() {
+    assert_parse_error_human(
+        &b2_human(B3_human()),
+        &ExpectedErrorMessageBuilder::error("failed to resolve type: NS2::MyType")
+            .help("`NS2::MyType` has not been declared")
+            // TODO: why don't we have an underline here? (Uncommenting this produces test failure)
+            //.exactly_one_underline("NS2::MyType")
+            .build(),
+    );
+}
+#[test]
+fn B3c_human() {
+    assert_parse_error_human(
+        &c_human(B3_human()),
+        &ExpectedErrorMessageBuilder::error("failed to resolve type: NS2::MyType")
+            .help("`NS2::MyType` has not been declared")
+            // TODO: why don't we have an underline here? (Uncommenting this produces test failure)
+            //.exactly_one_underline("NS1::MyType")
+            .build(),
+    );
+}
+#[test]
+fn B3d1_human() {
+    assert_parses_successfully_human(&d1_human(B3_human()));
+}
+#[test]
+fn B3d2_human() {
+    assert_parses_successfully_human(&d2_human(B3_human()));
+}
+#[test]
+fn C1a1_human() {
+    assert_parses_successfully_human(&a1_human(C1_human()));
+}
+#[test]
+fn C1a2_human() {
+    assert_parses_successfully_human(&a2_human(C1_human()));
+}
+#[test]
+fn C1b1_human() {
+    assert_parses_successfully_human(&b1_human(C1_human()));
+}
+#[test]
+fn C1b2_human() {
+    assert_parses_successfully_human(&b2_human(C1_human()));
+}
+#[test]
+fn C1c_human() {
+    assert_parse_error_human(
+        &c_human(C1_human()),
+        &ExpectedErrorMessageBuilder::error("failed to resolve type: MyType")
+            .help("neither `NS1::MyType` nor `MyType` refers to anything that has been declared")
+            // TODO: why don't we have an underline here? (Uncommenting this produces test failure)
+            //.exactly_one_underline("MyType")
```

Comment:
Could be the same cause as #948

Reply:
I actually think it's #147.  See [this thread](https://github.com/cedar-policy/cedar/pull/1060#discussion_r1674552526)